stl/inc/__msvc_int128.hpp

```diff
@@ -43,14 +43,17 @@ _STD_BEGIN
 #if defined(_M_X64) && !defined(_M_ARM64EC) && !defined(_M_CEE_PURE) && !defined(__CUDACC__) \
     && !defined(__INTEL_COMPILER)
 #define _STL_128_INTRINSICS 1
-#ifdef __clang__ // clang doesn't have _udiv128 / _div128
-#define _STL_128_DIV_INTRINSICS 0
-#else // ^^^ Clang / other vvv
-#define _STL_128_DIV_INTRINSICS 1
-#endif // ^^^ detect _udiv128 / _div128 ^^^
+#ifdef __clang__
+#define _STL_128_ADD_SUB_INTRINSICS 0 // clang doesn't have _addcarry_u64 / _subborrow_u64

```

Comment:
Naming nit: these are 64-bit intrinsics, not 128-bit intrinsics.
More serious: This is a lie. I can reproduce the issue with the clang in 17.7p1, but not when I self-install 16.0.1. Something has gone awry in the VS clang tree, and we need to run it down.

I don't really care about the naming given that I suspect this will be a temporary workaround. I've filed #3713 to investigate this.